### DISCUSSION TITLE
Decimals in the form number field

### DISCUSSION THREAD

**Discussion by Gruv69** [elementor/elementor#24271] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I am unable to put number with decimals in the number field when filling the online form. 

For my case, I need people to put in their km or miles that they did during the day and it needs to be precise. Number field only accept numbers without decimals. 

I'm also using the Amount filed which can calculate all the number fields they filled. The amount field can already calculate with decimals which is great.

### Describe the solution you'd like

Allowing number field to put 2 decimals.

### Describe alternatives you've considered

Asked for a workaround solution on the FB community.

### Additional context

print screen of problem message.
![Screenshot 2023-11-13 092527](https://github.com/elementor/elementor/assets/125513391/5813bdac-d42b-4237-88b1-9e739523e2a8)
[https://projsoins.com/km/](url)

**Comment by mingjun1120** [comment]
@Gruv69, I am facing the same issue as you are now. Any solution for this?
